src/js/enactors.js

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.firstDiscovery.enactor");
+
+    /*
+     * Langauge enactor
+     */
+    fluid.defaults("gpii.firstDiscovery.enactor.lang", {
+        gradeNames: ["fluid.prefs.enactor", "autoInit"],
+        preferenceMap: {
+            "locale": {
```

Comment:
Unfortunately, the way the prefs framework to find the current language is to look up the central model for a fixed path "locale". We don't a mechanism to identify any string as a model path for the language. That's why it's not a namespaced term being used here, similar changes in the primary and aux schema for language.